README.md

```diff
@@ -8,7 +8,7 @@ This repository is intended to be used in conjunction with other parts of the te
 library (standard, os, grid...) to actually configure a system. To help putting together
 everything needed in a way directly usable in a real configuration, it is recommended to
 follow the procedure describe at 
-https://trac.lal.in2p3.fr/Quattor/wiki/Download/QWGTemplates/Install#Upgradinganexistingrepository.
+http://www.quattor.org/documentation/2014/06/06/how-to-use-template-library.html#how-to-get-it?.
```

Comment:
I'll do it, thanks for the suggestion. I originally didn't do it because I wanted to keep this file as a minimalistic markdown file for people reading it as a text file. But in this particular case, you are right, it will make the things clearer.